### ISSUE TITLE
Test failure: baseservices\\threading\\paramthreadstart\\ThreadStartUShort_2\\ThreadStartUShort_2.cmd

### ISSUE THREAD

**Issue by v-haren** [dotnet/runtime#45098]
failed in job: [runtime-coreclr ilasm 20201121.1 ](https://dev.azure.com/dnceng/public/_build/results?buildId=895578&view=ms.vss-test-web.build-test-results-tab&runId=28621456&resultId=103575&paneView=debug)

CoreCLR windows arm64 Checked ilasmroundtrip @ Windows.10.Arm64v8.Open

Error message
~~~
Return code:      1
Raw output file:      D:\h\w\B02409B0\w\9404088B\e\baseservices\threading\Reports\baseservices.threading\paramthreadstart\ThreadStartUShort_2\ThreadStartUShort_2.output.txt
Raw output:
BEGIN EXECUTION
 cmd /c ..\ThreadStartUShort\ThreadStartUShort.cmd
BEGIN EXECUTION
D:\h\w\B02409B0\p\ildasm.exe /raweh /out=ThreadStartUShort.dasm.il ThreadStartUShort.dll
D:\h\w\B02409B0\p\ilasm.exe /output=ThreadStartUShort.asm.dll  ThreadStartUShort.dasm.il

Microsoft (R) .NET IL Assembler.  Version 6.0.0-ci
Copyright (c) Microsoft Corporation.  All rights reserved.
Assembling 'ThreadStartUShort.dasm.il'  to EXE --> 'ThreadStartUShort.asm.dll'
Could not open ThreadStartUShort.dasm.il

***** FAILURE ***** 
EXECUTION OF ILASM - FAILED 1
Expected: 0
Actual: 1
END EXECUTION - FAILED
FAILED
Test Harness Exitcode is : 1
To run the test:
> set CORE_ROOT=D:\h\w\B02409B0\p
> D:\h\w\B02409B0\w\9404088B\e\baseservices\threading\paramthreadstart\ThreadStartUShort_2\ThreadStartUShort_2.cmd
Expected: True
Actual:   False


Stack trace
   at baseservices_threading._paramthreadstart_ThreadStartUShort_2_ThreadStartUShort_2_._paramthreadstart_ThreadStartUShort_2_ThreadStartUShort_2_cmd() in F:\workspace\_work\1\s\artifacts\tests\coreclr\windows.arm64.Checked\TestWrappers\baseservices.threading\baseservices.threading.XUnitWrapper.cs:line 34393
~~~

**Comment by BruceForstall** [comment]
Most likely some kind of infrastructure issue due to file locking.

**Comment by briansull** [comment]
Probably a duplicate of #44139

**Comment by briansull** [comment]
Fixed by #44138
